src/modules/wishlist/components/wishlist.tsx

```diff
@@ -34,11 +35,12 @@ export function Wishlist() {
             <AccommodationCard
               status={status}
               id={item.accommodation.id}
-              image={CardImage}
+              image={item.accommodation.coverImage ? item.accommodation.coverImage : DefaultImage}
               name={item.accommodation.name}
               address={item.accommodation.address}
               pricePerNight={item.accommodation.price}
-              rating={4.8}
+              isSavedToWishlist={true}
```

Comment:
I think we need to add isSavedToWishlist to the wishlist response type as well so we can track changes. Currently, it won't remove the accommodation until the user refreshes the page.

Reply:
you know, i already declared it in Accommodation type, but it's still  there. I think this is not a bug, it's a feature))
User might accidentally press the heart button, then they have a chance to make it back)